packages/drivers/routerlicious-driver/src/test/wholeSummaryDocumentStorageService.spec.ts

```diff
@@ -0,0 +1,149 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+/* eslint-disable max-len */
+
+import assert from "assert";
+import { DebugLogger } from "@fluidframework/telemetry-utils";
+import {
+    SummaryType,
+} from "@fluidframework/protocol-definitions";
+import {
+    IWholeFlatSummary,
+    IWholeFlatSummaryBlob,
+    IWholeFlatSummaryTreeEntry,
+} from "@fluidframework/server-services-client";
+import { WholeSummaryDocumentStorageService } from "../wholeSummaryDocumentStorageService";
+
+const summaryBlobs: IWholeFlatSummaryBlob[] = [
+    {
+        id: "bARCTBK4PQiMLVK2gR5hPRkId",
+        content: "[]",
+        encoding: "utf-8",
+        size: 2,
+    },
+    {
+        id: "bARCfbIYtOyFwf1+nY75C4UFc",
+        content: "{\"createContainerRuntimeVersion\":\"0.59.1000\",\"createContainerTimestamp\":1651351060440,\"summaryFormatVersion\":1,\"gcFeature\":0}",
+        encoding: "utf-8",
+        size: 125,
+    },
+    {
+        id: "bARAL2CXvHYOch_aQtJAJOker",
+        content: "[[\"code\",{\"key\":\"code\",\"value\":{\"package\":\"no-dynamic-package\",\"config\":{}},\"approvalSequenceNumber\":0,\"commitSequenceNumber\":0,\"sequenceNumber\":0}]]",
+        encoding: "utf-8",
+        size: 149,
+    },
+];
+
+const treeEntries: IWholeFlatSummaryTreeEntry[] = [
+    {
+        path: ".protocol",
+        type: "tree",
+    },
+    {
+        id: "bARCTBK4PQiMLVK2gR5hPRkId",
+        path: ".protocol/attributes",
+        type: "blob",
+    },
+    {
+        id: "bARAL2CXvHYOch_aQtJAJOker",
+        path: ".protocol/quorumValues",
+        type: "blob",
+    },
+    {
+        path: ".app",
+        type: "tree",
+    },
+    {
+        path: ".app/.channels",
+        type: "tree",
+    },
+    {
+        path: ".app/.channels/rootDOId",
+        type: "tree",
+    },
+    {
+        id: "bARCfbIYtOyFwf1+nY75C4UFc",
+        path: ".app/.metadata",
+        type: "blob",
+    },
+];
+
+const flatSummary: IWholeFlatSummary = {
+    id: "bBwAAAAAHAAAA",
+    trees: [
+        {
+            id: "bBwAAAAAHAAAA",
+            sequenceNumber: 0,
+            entries: treeEntries,
+        },
+    ],
+    blobs: summaryBlobs,
+};
+
+const iSummary = {
```

Comment:
Can you please add a comment saying this is the expected summary that is returned by `downloadSummary`.
Also, rename this to expectedSummary? And add the ISummaryTree type?